ui/.gitignore

```diff
@@ -18,3 +18,4 @@
 # Binaries for programs and plugins
 dist
 node_modules
+ui-licenses
```

Comment:
What is this folder about?

Reply:
a temp folder for UI-related license files, the build script will remove it eventually.